bindsnet/pipeline/__init__.py

```diff
@@ -75,17 +75,20 @@ def __init__(self, network: Network, environment: Environment, encoding: Callabl
         else:
             self.history = {}
 
+        # monitor spikes for selecting action based on sum of spikes
```

Comment:
It seems like this shouldn't be done in the `Pipeline` object, but outside in an experimental script. If the layers of the network are really large, for example, this could make things really slow.

Reply:
Yes. You are right. I will make the change.